spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1985,6 +2013,27 @@ def hide_tooltip(self):
         self.tooltip_widget.hide()
         self.clear_extra_selections('code_analysis_highlight')
 
+    # --- Hint for completions
+    def show_hint_for_completion(self, word, documentation, at_point):
+        # Change to use signal from list widget

```

Comment:
@dalthviz what is this comment doing here above the docstring? Is it a TODO?

Reply:
@goanpeca yep it was a TODO